qemu/tests/cfg/ple_test.cfg

```diff
@@ -0,0 +1,18 @@
+- ple_test:
+    virt_test_type = qemu
+    type = ple_test
+    not_preprocess = yes
```

Comment:
I have some concern about this line and L55 `env_process.preprocess_vm(test, params, env, params["main_vm"])`.

With `not_preprocess = yes`, both `preprocess_image` and `preprocess_vm` will be skipped. Later with L55, only `preprocess_vm` will be executed. so i suggest the following two ways:

1. `not_preprocess = yes`   ->  `env_process.process(test, params, env, env_process.preprocess_image,env_process.preprocess_vm)`


2. `start_vm = no` - > `start_vm = yes`, `env_process.preprocess_vm`

Reply:
Thanks for all the details. I selected the second one. And test passed. Thanks.